### PR TITLE
Add default translations for io add-ons

### PR DESCRIPTION
**Pull request by wborn**
This makes the texts used by these add-ons translatable with Crowdin.

### CODE REVIEW

**Review by yfre — APPROVED** [review]
looks good for homekit

File: bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+io.config.neeo.checkStatusInterval.label = Check Status Interval (seconds)
+io.config.neeo.checkStatusInterval.description = The interval (in seconds) to check the status of the brain
+io.config.neeo.exposeAll.label = Expose All
+io.config.neeo.exposeAll.description = Expose all things
+io.config.neeo.exposeNeeoBinding.label = Expose NEEO Binding
+io.config.neeo.exposeNeeoBinding.description = Expose things found by the NEEO Binding
+io.config.neeo.searchLimit.label = Search Limit
+io.config.neeo.searchLimit.description = The maximum number of results to return for a search
```
Comment (lolodomo) [inline bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties:8]: We don't have an entry for the binding label

**Review by lolodomo — CHANGES_REQUESTED** [review]
Note that there is nothing for the ImperiHome integration service.

File: bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties (hunk 1 above)
Comment (wborn) [reply bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties:8]: I have made a fix for it @lolodomo!

**Comment by wborn** [comment]
> Note that there is nothing for the ImperiHome integration service.

It doesn't seem to have any configuration parameters:

https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.io.imperihome/src/main/resources/OH-INF/config/config.xml

But it is a configurable service:

https://github.com/openhab/openhab-addons/blob/cafed142e7dfac916eda8e6d893b5817a4a89e0b/bundles/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/ImperiHomeApiServlet.java#L66

Looks like it can all be removed and readded when neccessary in the future?

**Comment by lolodomo** [comment]
Ok, so we can probably forget Imperihome.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**